css-overflow-4/Overview.bs

```diff
@@ -1140,6 +1163,44 @@ Fragmentation of Overflow: the 'continue' property</h3>
 	as the box would be a [=flex container=] rather than a [=block container=],
 	and thus the 'continue' property would not apply.
 
+<h4 id=line-clamp-containers>Line-clamp containers</h4>
+
+	A <dfn>line-clamp container</dfn> is an [=independent formatting context|independent=] [=block formatting context=] that
+	additionally follows the rules in this section. All line-clamp containers contain a
+	<dfn>clamp point</dfn>, which is one of the following positions inside it:
+
+	- A point immediately after a [=line box=] in the line-clamp container's [=block formatting
+	  context=].
+	- The end of the line-clamp container, if there are any boxes after the last [=inline formatting
+	  context=] in the line-clamp container's [=block formatting context=].
+
+	If the [=line-clamp container=]'s [=block formatting context root=] has a [=computed value=] of
+	'max-lines' other than ''max-lines/none'', then that property will determine the [=clamp point=].
+	Otherwise, the clamp point will be set to the last possible clamp point such that, for it and
+	all previous possible clamp points, the line-clamp container's [=automatic block size=] (as
+	determined below) is not greater than the [=block size=] the box would have if its automatic
+	block size were infinite; or if that is not the case for any clamp points, to the first clamp
+	point in the block formatting context.
+
+	Any boxes in a [=line-clamp container=] that follow its [=clamp point=] in the box tree, as well
+	as any [=line boxes=] that follow it inside an [=inline formatting context=], will be invisible
+	(behaving like ''visibility: hidden''). This includes [=independent formatting contexts=] as
+	well as [=out-of-flow=] boxes, and all of their descendants. Any overflow such boxes and line
+	boxes might have is always counted as [=ink overflow=] rather than [=scrollable overflow=].
+
+	NOTE: This differs from the ''display: none''-like behavior of not rendered content with
+	''continue: discard''.
+
+	If a [=block container=] contains a [=clamp point=], within itself or in any of its descendants,
+	its [=automatic block size=] will not take into account any of its children after the clamp point.
+	This also applies for the [=line-clamp container=] itself.
+
+	NOTE: If there are any floats before the [=clamp point=], the [=line-clamp container=]'s
+	automatic size must grow to encompass the clearance, just like it would if it were a regular
+	[=block formatting context=] root that only contained its contents before the [=clamp point=].
+	This is regardless of whether this would cause any content after the [=clamp point=] to be
+	within the container's bounds.
```

Comment:
Editorial:
```suggestion
	NOTE: If there are any floats before the [=clamp point=],
	the [=line-clamp container=]'s automatic size must grow to encompass the clearance,
	just like it would if it were a regular [=block formatting context=] root
	that only contained its contents before the [=clamp point=].
	This is regardless of whether this would cause any content after the [=clamp point=]
	to be within the container's bounds.
```
See https://rhodesmill.org/brandon/2012/one-sentence-per-line/